src/components/organisms/plans/plans.scss

```
@@ -0,0 +1,39 @@
+@use "../../variables" as *;
+.plans {
+  padding:56px 0 100px 0;
+  @media (max-width: $breakpoint-md) {
+    padding: 56px 0 35px 0;
+  }
```

```suggestion
.plans {
  padding: 56px 0 100px 0;
  @media (max-width: $breakpoint-md) {
    padding: 56px 0 35px 0;
  }
```
このあたりフォーマッタが効いていない雰囲気があるので、 prettierの設定を見直してみてください。

参考動画：https://youtu.be/ijZIyqzHKeM?si=decVaFzX0_-NvBN4&t=313